core/arch/arm/plat-mediatek/platform_config.h

```diff
@@ -105,6 +105,20 @@
 #define CONSOLE_BAUDRATE	115200
 #define CONSOLE_UART_CLK_IN_HZ	26000000
 
+#elif defined(PLATFORM_FLAVOR_mt7988)
+
+#define GIC_BASE		0x0C000000
+#define GICC_OFFSET		0x00000
+#define GICD_OFFSET		0x00000
```

Comment:
1 of the offset is wrong

Reply:
Thanks your review. 
I fixed it. 

Thanks !!